### PR TITLE
Book detail 6 redirection

### PR DESCRIPTION
**Pull request by FedeMacias**
## Summary
- Added an hiperlink that takes you to the 'book-site.html' site using an icon and text.
- The icon was added using pseudo elements.
- The hiperlink works by clicking either the icon or the text.

## Snapshot
book-description-6-redirection

## Trello
https://trello.com/c/8iPR2Cih/33-detalle-de-libro-vi-redirecci%C3%B3n

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    flex-direction: row;
+    color: $black;
+    font-size: 20px;
+    line-height: 24px;
+    position: absolute;
+    top: 46px;
+    right: 1042px;
+    bottom: 702px;
+    left: 265px;
+    content: "book-list.html";
```
Comment (MMarussi) [inline scss/detail.scss:26]: Is this rule necessary?

File: scss/detail.scss [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+  align-items: center;
+
+  .return-button{
+    display: flex;
+    flex-direction: row;
+    color: $black;
+    font-size: 20px;
+    line-height: 24px;
+    position: absolute;
+    top: 46px;
```
Comment (MMarussi) [inline scss/detail.scss:22]: Remember that you can set `position: relative` on a parent of this element to simplify the positioning values :wink:

File: scss/detail.scss [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+    right: 1042px;
+    bottom: 702px;
+    left: 265px;
+    content: "book-list.html";
+  }
+
+  .return-button::before{
+    margin-right: 12px;
+    content: url(https://trello-attachments.s3.amazonaws.com/607457e8ad4162814da41ef0/607457e8ad4162814da41f0a/x/e42f9110cec094df3e49702baa8263f7/back-arrow-20.png);
```
Comment (MMarussi) [inline scss/detail.scss:31]: You can download the asset to your project and reference it from there

**Review by MMarussi — CHANGES_REQUESTED** [review]

File: scss/detail.scss [hunk 4]
```diff
@@ -7,12 +7,27 @@
...
   .cover-text-container {
+    position: relative;
     display: flex;
     height: 416px;
     width: 900px;
     background-color: $white;
     box-shadow: 4px 4px 10px 0 $black-rgba;
 
+    .return-button {
+      display: flex;
```
Comment (MMarussi) [inline scss/detail.scss:18]: Since ".return-button" element is not containing any element this rule can be removed

File: scss/detail.scss [hunk 5]
```diff
@@ -7,12 +7,27 @@
...
+      position: absolute;
+      color: $black;
+      font-size: 20px;
+      line-height: 24px;
+      top: -108px;
```
Comment (MMarussi) [inline scss/detail.scss:23]: I believe that you can use a margin to achieve this positioning without having to use `position: absolute`

File: book-list.html [hunk 6]
```diff
@@ -10,6 +10,15 @@
 </head>
 <body>
   <div class="master-container">
+    <div class="navbar-container">
+      <div class="navbar">
+        <img class="wolox-logo"
+          src="Images/LogoWolox.png" 
+          alt="wolox logo"
+        >
+        <p class="logout">Logout</p>
```
Comment (mcallegari10) [inline book-list.html:19]: Should be a button

File: scss/book-list-detail.scss [hunk 7]
```diff
@@ -1,12 +1,49 @@
...
+    top: 0;
+    left: 0;
+    border-top: 6px solid $cerulean; 
+    height: 70px;
+    width: 100%;
+    max-width: 1366px;
+    background-color: $white;
+    box-shadow: 0 4px 10px 0 $black-rgba;
+    align-items: center;
+    align-content: center;
```
Comment (mcallegari10) [inline scss/book-list-detail.scss:19]: Not needed 😄

File: scss/book-list-detail.scss [hunk 8]
```diff
@@ -1,12 +1,49 @@
...
+    .navbar {
+      display: flex;
+      height: 70px;
+      width: 100%;
+      max-width: 900px;
+      align-items: center;
+      align-content: center;
+      justify-content: space-between;
+      padding-left: 20px;
+      padding-right: 20px;
```
Comment (mcallegari10) [inline scss/book-list-detail.scss:31]: Use the shorthand

File: scss/book-list-detail.scss [hunk 9]
```diff
@@ -1,12 +1,49 @@
...
+    align-content: center;
+    justify-content: space-around;
+
+    .navbar {
+      display: flex;
+      height: 70px;
+      width: 100%;
+      max-width: 900px;
+      align-items: center;
+      align-content: center;
```
Comment (mcallegari10) [inline scss/book-list-detail.scss:28]: Jus `align-items` is needed

File: scss/detail.scss [hunk 10]
```diff
@@ -1,12 +1,28 @@
...
+  .return-button {
+    align-self: flex-start;
+    color: $black;
+    font-size: 20px;
+    line-height: 24px;
+    margin-bottom: 108px;
+    margin-left: 233px;
+  }
+
+  .return-button::before {
```
Comment (mcallegari10) [inline scss/detail.scss:19]: For pseudo-elements, use nesting and `&` operator:

```scss
.return-button {
  ...
  &::before {
    ...
  }
}
```

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: scss/detail.scss (hunk 10 above)
Comment (FedeMacias) [reply scss/detail.scss]: All changes made as requested!!

**Review by MMarussi — APPROVED** [review]

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**